styles/designs/webview/parts/_code-shapes.scss

```diff
@@ -29,3 +29,15 @@
       $CodeBlock--Centered,
     )
 ));
+
+@include create_shape('webview:::CodeSnippetLineNumbersShape', (
+  _components: (
+    $CodeSnippetLineNumbers,
+  ),
+));
+
+@include create_shape('webview:::CodeSnippetLineNumbersCodeShape', (
+  _components: (
+    $CodeSnippetLineNumbersCode,
+  ),
+));
```

Comment:
@kswanson33 When consolidating to one shape I get several errors.

Reply:
makes sense! `font-family` is a `REQUIRED` setting for this component, and having moved the component into a different shape, the way to set that setting may have changed. that would mean that at compile time, the compiler is looking for a `REQUIRED` setting to be set, and not finding it, so it gives an error.

regarding whether/why to have these components be in their own shape, or to put them both in one, maybe i should be asking you! here's context on how i'd think about that decision:

- components hold the properties for elements, while shapes are for features that might be made up of one or more elements
- some shapes only have a single component, because the feature they're styling only has one element.
- when a feature has multiple elements (like note, example, figure ...) we want to bring _all_ the styles for that feature into the book in _one_ `@import` statement, so nothing gets left behind
- what's the modularity of the "code block w/ line numbers" feature? would a book ever have just the line numbers, without the code block? what about just the code block without the line numbers?
- if these are separate features: the components can stay in different shapes
- if they always go together: the components should be in the same shape

Reply:
Thanks for the feedback and thorough explanation! I've consolidated the shapes for code snippets.